### ISSUE TITLE
Non-identical output given for same input

### ISSUE THREAD

**Issue by jxzhangcc** [OUnke/SpookyNet#7]
I noticed some weird error distribution when benchmarking this model. I finally figured out that spookynet.energy_and_forces does not always return the same output for the same input, unless I manually seed everything.
In particular, the output distribution is not Gaussian-like. Instead, the model has a ~90% probability to return value A and ~10% probability to return B or C (which diff from A by an amount far beyond numerical error), making this issue extremely difficult to capture.
The spookynet_calculator, however, does return same output for same input. Where does this issue come from? What's the difference between spookynet itself and spookynet calculator?

**Comment by jxzhangcc** [comment]
Turns out spookynet has a built-in module dropout mechanism which can be yet has to be manually turned off by model.eval()
